sos/plugins/openstack_instack.py

```diff
@@ -62,29 +63,34 @@ def setup(self):
                                 "the environment file for the user intended "
                                 "to connect to the OpenStack environment.")
         else:
+            # capture all the possible stack uuids
+            get_stacks = "openstack stack list"
+            stacks = self.call_ext_prog(get_stacks, timeout=600)['output']
```

Comment:
If this is set to `timeout=600` and can be expected to ever take that long, we'll need to adjust the plugin's timeout. Otherwise, the timeout of 600 seconds is longer than the default timeout for the whole plugin (which is 300).